source/getting_started/aiot_intelligent_edge/coco_for_applications/end_user_client_applications/android_end_user_client_applications/pre_install_checklist_android_end_user_client_apps.rst

```diff
@@ -3,9 +3,23 @@
 Pre-Install Checklist
 =====================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
+.. sectionauthor:: Krishna
 
-   **Coming Soon**
+The following are necessary to try out the **IoT SDK Android** and **Sample App**:
 
-.. sectionauthor:: Krishna
+Git
+    - To clone the Sample App repo from Github.
+    - `Git Installation <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+
+Android Studio
+    - To run the Sample App on an Android device or an emulator.
+    - `Android Studio Installation <https://developer.android.com/studio/install>`_
+
+Generating Client ID and Redirect URI
+    - If not using third-party login, the Client ID and Redirect URI are essential for a developer to handle authentication of his user.
+    - The steps for generation are found in :ref:`Manage the Buzz <native_apps>`.
+
+Setup Device App
+    - Device app is a gateway that comes with a **Network** which contains a **Resource** for demo purpose.
```

Comment:
The Device App doesn't come with a network. The network has to be created by the user using the COCO App and the software gateway added to it. Those steps will be covered in the Device App Quick Start Guide.

Reply:
corrected.